project/src/main/java/com/google/sps/servlets/AuthServlet.java

```diff
@@ -22,17 +22,55 @@
 import com.google.gson.Gson;
 import com.google.sps.data.*;  
 
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.Connection;
+import java.sql.SQLException;
+import javax.sql.DataSource;
+
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
 @WebServlet("/authentication")
 public class AuthServlet extends HttpServlet {
 
+  private static final Logger LOGGER = Logger.getLogger(AuthServlet.class.getName());
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    User userObj = User.create(1896,"Phoebe Khokgawe","phoebek@");
+    DataSource pool = (DataSource) request.getServletContext().getAttribute("my-pool");
```

Comment:
Maybe change `my-pool` to something more descriptive like `db-connection-pool` ?

Reply:
Done